src/com/untamedears/realisticbiomes/persist/WorldID.java

```diff
@@ -19,27 +24,53 @@ public static void init (Plugin plugin) {
 		
 		World end = plugin.getServer().getWorld("world_the_end");
 		WorldID.endID = end == null ? null : end.getUID();
+		
+		for (World world: plugin.getServer().getWorlds()) {
+			if (!world.getName().equals("world")
+					&& !world.getName().equals("world_nether")
+					&& !world.getName().equals("world_the_end")) {
+				if (otherWorlds == null) {
+					RealisticBiomes.doLog(Level.WARNING, "There are worlds that don't have default names (world, world_nether, world_the_end), experimental feature");
+					otherWorlds = new HashMap<UUID, Integer>();

```

Comment:
It is worth mentioning, however, that your lookup calls don't check if otherWorlds is initialized or not. Would be safer to always initialize -- defensive programming.